includes/multi-currency/Compatibility/WooCommerceSubscriptions.php

```diff
@@ -148,7 +148,7 @@ public function override_selected_currency( $return ) {
 
 		$subscription_renewal = $this->cart_contains_renewal();
 		if ( $subscription_renewal ) {
-			return get_post_meta( $subscription_renewal['subscription_renewal']['renewal_order_id'], '_order_currency', true );
+			return $subscription_renewal['subscription_renewal']['renewal_order_id']->get_currency();
```

Comment:
`$subscription_renewal['subscription_renewal']['renewal_order_id']` is an order ID/integer which doesn't have a `get_currency()` method.

We'll have to get the order object first, i.e.:
```
$order = wc_get_order( $subscription_renewal['subscription_renewal']['renewal_order_id'] );
return $order ? $order->get_currency() : false;
```

Reply:
D'oh! Of course! Fixed in 20ea6b0796fb535adac713e287c63caad100055e

Reply:
Note in all these I'm returning `$return` instead of hard-coded "false". Should be equivalent, but feels more in-keeping with the function comment. 

Side note – I don't really understand what this function is doing. The early return when passed `$return = false` param makes no sense to me, why call a function that can be disabled by passing false? 

https://github.com/Automattic/woocommerce-payments/blob/e01a19cab295bc982485642ca4d626b6b34c6728/includes/multi-currency/Compatibility/WooCommerceSubscriptions.php#L136-L147

We can dig into that when testing, I'm still figuring out how to test this 🤔